eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration.php

```diff
@@ -555,6 +555,7 @@ function ($v) {
                                     ->end()
                                     ->arrayNode('choices')
                                         ->scalarPrototype()->end()
+                                        ->performNoDeepMerging()
```

Comment:
I've just added this fix. It makes overriding of choices list possible and thus configuration more flexible.

Use case: if we ship later on Custom Tags `ezyoutube` and `eztwitter`, we'd like to define `css_class` or `align` attribute (e.g. as `['left', 'right', 'center']`).

End-Developer might want to change them w/o redefining the whole custom tag in the following way:
```yml
        custom_tags:
            ezyoutube:
                attributes:
                    align:
                        type: 'choice'
                        required: false
                        default_value: ''
                        choices: ['', 'class1', 'class2']
```
Result before the fix: `['', 'class1', 'class2', 'left', 'right', 'center']`
after the fix: `['', 'class1', 'class2']`

cc Reviewers @andrerom @adamwojs @bdunogier @webhdx @Nattfarinn